README.md

```diff
@@ -31,6 +31,12 @@ The ReSim CLI authenticates using client credentials (a client ID and a client s
 Client credentials can be specified on the commandline with the `--client_id` and `--client_secret` flags, or in the environment as
 `RESIM_CLIENT_ID` and `RESIM_CLIENT_SECRET`.
 
+If you would like to store your client ID and secret in a config file, the CLI will load them from `~/.resim/resim.yaml`.  The file
+is formatted as follows:
+
+    client_id: <client ID>
```

Comment:
This looks great. Happy to work with this as is for now. But I offer bonus kudos for building in some forward looking structure. For example will this file only hold credential? If so maybe it should be called `resim_credentials.yaml` alternatively, do we expect the file to be a catch all for more ReSim config? If so maybe the id and secret should be in some section/namespace called `credentials:`

Reply:
The structure of this file (right now, at least) is actually determined by Viper - it's just a collection of Viper flags.